go/array_string/two_sum/solution_test.go

```diff
@@ -0,0 +1,29 @@
+package solution
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestTwoSum(t *testing.T) {
+	cases := []struct {
+		name   string
+		input  []int
+		target int
+		want   []int
+	}{
+		{"with array elements that match the target", []int{0, 2, 4, 6, 7}, 13, []int{3, 4}},
+		{"with array elements that does not sum to target", []int{0, 2, 4, 6, 7}, 20, []int{}},
```

Comment:
You can use `nil` for your want value as well

Reply:
Yes, I just figured that out too, thanks.